product-features/liquidity-providers/how-to.md

```diff
@@ -0,0 +1,223 @@
+
+When **removing liquidity**, you will transfer your LP tokens back to the ALEX smart contract and receive your original funds plus any fees accrued while holding those LP tokens.
+
+## :palm\_up\_hand: :moneybag: Adding Liquidity
+
+### Step 1
+
+Go to [https://app.alexlab.co/](https://app.alexlab.co/) and click on navbar's Swap -> Pool tab.
+
+<figure><img src="../../.gitbook/assets/liquidity-providers/adding-liquidity-1-pool-tab.png" alt="" width="375"><figcaption></figcaption></figure>
+
+### Step 2
+
+All available pools will be displayed including information such as:
+
+* **Trading Pair:** The token pair that constitute liquidity pools to which you can add liquidity.
+* **Liquidity:** The total liquidity in the pool, expressed in USD value.
+* **Volume:** The trading volume between the token pair over the last 7 days.
+* **Fee Rebate:** Potential LP earnings from swap fees over a year, based on the last week's average. This metric, also known as Pool APR, reflects the potential profitability of participating in a pool over a year, assuming similar trading activity continues.&#x20;
+
+Select the token pair to which you want to add liquidity from the displayed list. Note you can sort by pool metrics.
+
+<figure><img src="../../.gitbook/assets/liquidity-providers/adding-liquidity-2-select-pool.png" alt=""><figcaption><p>Selected STX - ALEX liquidity pool as example.</p></figcaption></figure>
+
+{% hint style="warning" %}
+When hovering over a pool, you might notice a "+LP" button. This is just a visual indicator highlighting the selected pool. Clicking it will take you to the same screen as clicking anywhere on the pool's row.
+{% endhint %}
+
+### Step 3
+
+After selecting a pool, you will be taken to a control panel for that specific liquidity pool, where you can add liquidity to the token pair and view more detailed metrics.&#x20;
+
+When you set the amount for one token, the corresponding amount for the other token is automatically calculated, as liquidity must be provided in equal value for both tokens.
+
+<figure><img src="../../.gitbook/assets/liquidity-providers/adding-liquidity-3-pool-control-panel.png" alt=""><figcaption><p>Control panel example for STX - ALEX liquidity pool. Amount is set to 4 STX and ALEX token amount is automatically determined.</p></figcaption></figure>
+
+{% hint style="info" %}
+Clicking the "Max" button will automatically set the amount to your total available balance.
+{% endhint %}
+
+{% hint style="warning" %}
+The APR metric is the same as the displayed in the Fee Rebate column on the previous step.
+{% endhint %}
+
+### Step 4
+
+If you want to adjust slippage, select the "Settings" icon to open the Transaction Settings panel and set your desired tolerance. The default slippage tolerance is set to 4%, meaning your transaction will revert if the price changes unfavourably by more than this percentage. The displayed number of LP tokens you will receive is approximate due to this potential variation.
```

Comment:
If you want to adjust slippage, select the "Settings" icon to open the Transaction Settings panel and set your desired tolerance. The default slippage tolerance is set to 4%, meaning your transaction will revert if the exchange rate changes unfavorably by more than this percentage. The displayed number of LP tokens you will receive is an approximation due to this potential variation.

Reply:
Done